### PR TITLE
Exceptions Folder 

### PR DESCRIPTION
**Pull request by karthikamurthy**
Contains all the exceptions related to direct and proxy

### CODE REVIEW

File: src/internal/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (rranjan3) [inline src/internal/__init__.py:1]: Add copyright and License statement.

File: src/internal/exceptions/invalid_parameter.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enums.error_code import JRPCErrorCodes
+
+class InvalidParamException(Exception):
+    """All errors and status are returned in
```
Comment (rranjan3) [inline src/internal/exceptions/invalid_parameter.py]: Update comment to reflect only meant for Invalid parameter types of errors.

File: src/internal/exceptions/proxy/contract_error.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enums.proxy.contract_response import ContractResponse
+
+class ContractFailed(Exception):
+    """All errors and status are returned in
```
Comment (rranjan3) [inline src/internal/exceptions/proxy/contract_error.py]: Here as well. Reword to specific case.

**Review by rranjan3 — CHANGES_REQUESTED** [review]

File: src/internal/__init__.py (hunk 1 above)
Comment (karthikamurthy) [reply src/internal/__init__.py:1]: done

File: src/internal/exceptions/invalid_parameter.py (hunk 2 above)
Comment (karthikamurthy) [reply src/internal/exceptions/invalid_parameter.py]: done

File: src/internal/exceptions/proxy/contract_error.py (hunk 3 above)
Comment (karthikamurthy) [reply src/internal/exceptions/proxy/contract_error.py]: done

**Review by rranjan3 — APPROVED** [review]

**Pull request merged**